src/test/java/seedu/address/model/loan/LoanListTest.java

```diff
@@ -43,11 +43,22 @@ public void contains() {
     }
 
     @Test
-    public void remove() {
-        LoanList loanList = new LoanList();
-        loanList.add(LOAN_1);
+    public void addToNewCopy() {
+        LoanList loanList1 = new LoanList();
+        loanList1.add(LOAN_1);
+        LoanList loanList2 = loanList1.addToNewCopy(LOAN_2);
+        loanList1.add(LOAN_2);
+        assertEquals(loanList1, loanList2);
+        assertFalse(loanList1 == loanList2);
```

Comment:
maybe we can use `assertNotEquals(loanList1, loanList2)` here to be consistent

Reply:
Updated to use assertNotSame